### ISSUE TITLE
Menu items are quite large and need to be smaller

### ISSUE THREAD

**Issue by ramblerswebs** [ramblerswebs/Gantry5_Template#2]
Size of menu items needs to be smaller, perhaps user controllable 
the menu on site https://test.derbyramblers.org.uk/ uses two lines on my laptop

**Comment by keithgrimes** [comment]
Font Size for the menu can now be overridden/set within the template style page.

**Comment by keithgrimes** [comment]
> Size of menu items needs to be smaller, perhaps user controllable
> the menu on site https://test.derbyramblers.org.uk/ uses two lines on my laptop

Chris, have updated the template and changed the menu size on your test site. Font size for menus can now be set within the style.

**Comment by ramblerswebs** [comment]
Sorry but cannot find where this option is. Where is it?
I went to Template >  Gantry 5 > style

I was also wondering if the spacing between menu items could be smaller.
If you compare the test site with https://derbyramblers.org.uk/ the menu takes a lot more space

**Comment by keithgrimes** [comment]
If you are on the style settings. There is a section further down for navigation and an entry for Font-Size.

**Comment by ramblerswebs** [comment]
I have tried that now, first time I did it nothing seemed to happen.
First question - what is the difference between the Component - Style and the Style under Templates? They seem the same to me.

I have edited the style, both component and template, and set the text size to 13px which is the same size as the production site derbyramblers.org.uk
You will see the two menus look different. The font sizes do not look the same to me. The G5 menu has a lot more space around the menu items 
The G5 default menu looks great but most web masters seem to end up with lots of menu items and struggle to keep them on a single line. So the extra spacing will make this worse.
Are there any options to reduce the spacing? Perhaps we need a **compact** option
